tests/test_click.py

```diff
@@ -11,6 +13,22 @@ def test_hello_world():
     assert "THM{" in result.output
 
 
+def test_filtration():
+    runner = CliRunner()
+    result = runner.invoke(main, ["--rarity", "0.5:", "--include_tags", "Identifiers,Media", "fixtures/file"])
+    assert result.exit_code == 0
+    assert "THM{" in result.output
+    assert "ETH" in result.output
+
+
+def test_tag_printing():
+    runner = CliRunner()
+    result = runner.invoke(main, "--tags")
+    assert result.exit_code == 0
+    for tag in pywhat_tags:
+        assert tag in result.output

```

Comment:
Do you think tags should be non-case sensitive? 
![image](https://user-images.githubusercontent.com/10378052/120917511-9d8c7c80-c6a7-11eb-9024-992b2b5b6dd7.png)